compose/compose-file/compose-versioning.md

```diff
@@ -227,6 +227,8 @@ several options have been removed:
     `deploy`. Note that `deploy` configuration only takes effect when using
     `docker stack deploy`, and is ignored by `docker-compose`.
 
+-   `extends`: This option has been removed for `version: "3"` Compose files.
```

Comment:
@londoncalling I think you missed this instance - afaik we don't use that notation anywhere in the docs, so it should be modified to version 3.x (especially because the comment applies to 3.1 as well)